addons/web_editor/static/src/js/wysiwyg/widgets/image_crop_widget.js

```diff
@@ -92,8 +92,9 @@ const ImageCropWidget = Widget.extend({
         this._onDocumentKeydown = this._onDocumentKeydown.bind(this);
         // We use capture so that the handler is called before other editor handlers
         // like save, such that we can restore the src before a save.
-        this.document.addEventListener('mousedown', this._onDocumentMousedown, {capture: true});
-        this.document.addEventListener('keydown', this._onDocumentKeydown, {capture: true});
+        this.ownerDocument = this.$el[0].ownerDocument;
```

Comment:
Right, so the media is in an iframe but the ui is out of the iframe. I would rename that to be clear on which document we're talking about. In fact since `this.el` also exists and `this.el.ownerDocument` just adds 3 characters, I just wouldn't name it at all and just type that in full, to be explicit wherever you need it.

With your changes, `this.document` seems not to be used anymore. But since we're in stable and this is a public constant, we can't juste remove it (in case anyone's custom code makes use of it). What I suggest instead is to replace it with a deprecated getter: `get function()` with the `@deprecated` tag in its doc string, returning the original value of `this.document` (namely, `this.media.ownerDocument`) and logging a warning.

Reply:
I just rebased the branch and found that in this recently merged [PR](https://github.com/odoo/odoo/pull/153684/files) (as well as in further versions)  `this.document` is being used. Do we really need to replace it by the getter function ?

Reply:
Ah good catch! Well in that case no let's not bother.

Reply:
cc @dmo-odoo fyi